packages/cache/src/stylis-plugins.js

```diff
@@ -101,14 +101,6 @@ export let compat = element => {
     element.column === parent.column && element.line === parent.line
 
   while (parent.type !== 'rule') {
-    if (
-      // @media
-      parent.type.charCodeAt(1) !== 109 &&
-      // @supports
-      parent.type.charCodeAt(1) !== 115
-    ) {
-      return
-    }
```

Comment:
This was a fix for `stylis@4.0.11` that was out briefly but Sultan has patched the "regression" in a timely manner, and released `stylis@4.0.12`. So I've decided not to include this fix.